commons-test/src/main/java/org/infinispan/commons/test/BlockHoundHelper.java

```diff
@@ -17,6 +17,7 @@ static void installBlockHound() {
       // This is a terrible hack but gets around the issue that blockhound doesn't allow the registering thread
       // to be a dynamic blocking thread - in which case our checks in
       // AbstractInfinispanTest#currentThreadRequiresNonBlocking will never be evaluated
+      // To be removed when https://github.com/reactor/BlockHound/issues/121 is fixed
```

Comment:
it looks like https://github.com/reactor/BlockHound/issues/121 is already fixed. you can remove this line :rofl:

Reply:
Sure :D